OMGVA PoS/Data layer/DTOs/Tax/CreateTaxRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace OMGVA_PoS.Data_layer.DTOs
+{
+    public class CreateTaxRequest
```

Comment:
Not sure how about the validation, but it may be an option to use just Tax model. I know that tax model also has id, but it could be left empty. That way there would be one class less. But not surre which one is better.

Reply:
Fine, let it be.